amida-tech/recruitment-registry
# enforce HTTP only for JWT tokens

#### What's this PR do?
Mark JWT cookies as HTTP only to prevent session hijacking.

#### Related JIRA tickets:
https://jira.amida-tech.com/browse/RR-695

#### How should this be manually tested?
Start the API locally, then open Swagger. Open the developer console, then submit an `/auth/basic` request using `Basic c3VwZXI6QW0hZEAyMDE3UFc=` for authorization. In the console network tab, select the request, then go to the cookies tab. You should see `rr-jwt-token` under response cookies, and the HTTP column should be checked.

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/954072/28575700-177a0c9e-7128-11e7-9df9-f6440b5e11f5.png)
